doc/devel/development_workflow.rst

```diff
@@ -6,41 +6,34 @@
 Development workflow
 ####################
 
-You already have your own forked copy of the `Matplotlib`_ repository, by
-following :ref:`forking`. You have :ref:`set-up-fork`. You have configured
-git by following :ref:`configure-git`.  Now you are ready for some real work.
-
 Workflow summary
 ================
 
 In what follows we'll refer to the upstream Matplotlib ``main`` branch, as
 "trunk".
```

Comment:
I think the term *trunk* was quite common in SVN, but is not in git. I suggest to use `main` or `upstream/main` and not introduce a new name.